exercises/practice/resistor-color/.docs/instructions.append.md

```diff
@@ -0,0 +1,5 @@
+# Instructions append
+
+WebAssembly does not have the concept of arrays. For the colors test, return a comma-delimited buffer of characters in the form:
```

Comment:
Is there something missing at the end of this line?

Reply:
Sure is! Good catch! 🙌